Astro binding: Null Pointer exception

Since around a couple of days the Astro binding is throwing a Null Pointer exception. It started already before moving the extension to eclipse smarthome.
Related community thread:
https://community.openhab.org/t/astrobinding-throws-an-nullpointer-error/27790
openhab.log:
```
2017-05-16 00:00:00.209 [ERROR] [org.quartz.core.JobRunShell         ] - Job astro:sun:shutterkitchen.job-daily-sun threw an unhandled Exception: 
java.lang.NullPointerException
	at org.eclipse.smarthome.binding.astro.internal.util.DateTimeUtils.applyConfig(DateTimeUtils.java:180)[187:org.eclipse.smarthome.binding.astro:0.9.0.201705120951]
	at org.eclipse.smarthome.binding.astro.internal.job.AbstractDailyJob.scheduleEvent(AbstractDailyJob.java:71)[187:org.eclipse.smarthome.binding.astro:0.9.0.201705120951]
	at org.eclipse.smarthome.binding.astro.internal.job.AbstractDailyJob.scheduleRange(AbstractDailyJob.java:58)[187:org.eclipse.smarthome.binding.astro:0.9.0.201705120951]
	at org.eclipse.smarthome.binding.astro.internal.job.DailyJobSun.schedulePlanetEvents(DailyJobSun.java:35)[187:org.eclipse.smarthome.binding.astro:0.9.0.201705120951]
	at org.eclipse.smarthome.binding.astro.internal.job.AbstractDailyJob.executeJob(AbstractDailyJob.java:47)[187:org.eclipse.smarthome.binding.astro:0.9.0.201705120951]
	at org.eclipse.smarthome.binding.astro.internal.job.AbstractBaseJob.execute(AbstractBaseJob.java:40)[187:org.eclipse.smarthome.binding.astro:0.9.0.201705120951]
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)[105:org.eclipse.smarthome.core.scheduler:0.9.0.201705120951]
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)[105:org.eclipse.smarthome.core.scheduler:0.9.0.201705120951]
2017-05-16 00:00:00.218 [ERROR] [org.quartz.core.ErrorLogger         ] - Job (astro:sun:shutterkitchen.job-daily-sun threw an exception.
org.quartz.SchedulerException: Job threw an unhandled exception. [See nested exception: java.lang.NullPointerException]
	at org.quartz.core.JobRunShell.run(JobRunShell.java:213)[105:org.eclipse.smarthome.core.scheduler:0.9.0.201705120951]
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)[105:org.eclipse.smarthome.core.scheduler:0.9.0.201705120951]
Caused by: java.lang.NullPointerException
	at org.eclipse.smarthome.binding.astro.internal.util.DateTimeUtils.applyConfig(DateTimeUtils.java:180)[187:org.eclipse.smarthome.binding.astro:0.9.0.201705120951]
	at org.eclipse.smarthome.binding.astro.internal.job.AbstractDailyJob.scheduleEvent(AbstractDailyJob.java:71)[187:org.eclipse.smarthome.binding.astro:0.9.0.201705120951]
	at org.eclipse.smarthome.binding.astro.internal.job.AbstractDailyJob.scheduleRange(AbstractDailyJob.java:58)[187:org.eclipse.smarthome.binding.astro:0.9.0.201705120951]
	at org.eclipse.smarthome.binding.astro.internal.job.DailyJobSun.schedulePlanetEvents(DailyJobSun.java:35)[187:org.eclipse.smarthome.binding.astro:0.9.0.201705120951]
	at org.eclipse.smarthome.binding.astro.internal.job.AbstractDailyJob.executeJob(AbstractDailyJob.java:47)[187:org.eclipse.smarthome.binding.astro:0.9.0.201705120951]
	at org.eclipse.smarthome.binding.astro.internal.job.AbstractBaseJob.execute(AbstractBaseJob.java:40)[187:org.eclipse.smarthome.binding.astro:0.9.0.201705120951]
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)[105:org.eclipse.smarthome.core.scheduler:0.9.0.201705120951]
	... 1 more
```

--- a/extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/job/AbstractDailyJob.java
+++ b/extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/job/AbstractDailyJob.java
@@ -61,6 +61,10 @@ public abstract class AbstractDailyJob extends AbstractBaseJob {
 
     protected void scheduleEvent(String thingUid, AstroThingHandler astroHandler, Calendar eventAt, String event,
             String channelId) {
+        if (eventAt == null) {
+            logger.trace("Timestamp for job for event {} of channel {} was NULL", event, channelId);
+            return;
+        }
         JobDataMap jobDataMap = new JobDataMap();
         jobDataMap.put(KEY_THING_UID, thingUid);
         jobDataMap.put(EventJob.KEY_EVENT, event);

